docs/project_structure.md

```diff
@@ -1,14 +1,10 @@
 ### Project Structure
 
-The project's components are organized, conceptually, into 3 layers:
```

Comment:
Suggest reorganizing this section but keeping the gist of this content.

Reply:
Updated.